core/templates/dev/head/components/forms/ImageUploaderDirective.js

```diff
@@ -43,11 +45,24 @@ oppia.directive('imageUploader', [
 
         $(elt).bind('dragover', function(e) {
           e.preventDefault();
-          $(elt).addClass('image-uploader-is-active');
+          $('.image-uploader-drop-area').addClass('image-uploader-is-active');
         });
 
         $(elt).bind('dragleave', onDragEnd);
 
+        // If the user accidentally drops an image outside of the image-uploader
+        // we want to prevent the browser from applying normal drag-and-drop
+        // logic, which is to load the image in the browser tab.
+        $(window).bind('dragover', function(e) {
+          e = e || event;
```

Comment:
Some browsers (old browsers, according to stuff I've read) don't pass the event object as parameter, but instead put the object in a global variable (window.event or.. just event). So this is protection against that.

That being said, we agreed offline that we'll remove this, since it's not consistent with how we do it elsewhere.